modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -266,8 +271,22 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
 #ifdef HAVE_CUDA
         cudaFusionMode = cuda4dnn::ConvolutionConfiguration::FusionMode::NONE;
         cudaActType = cuda4dnn::ConvolutionConfiguration::ActivationType::IDENTITY;
+#endif
+#ifdef HAVE_TENGINE
+        graph=NULL;
+        tengine_ret = false ; //false = 0
 #endif
     }
+#ifdef HAVE_TENGINE
+    ~ConvolutionLayerImpl()
+    {
+        if(NULL != graph )
```

Comment:
I suggest to move this code with `postrun_graph` and `destroy_graph` into a separate function in `tengine_graph_convolution.h/.cpp`, to make API symmetric.

Reply:
ok. i will do this .